index.html

```diff
@@ -155,6 +157,80 @@ <h2>
         </p>
       </div>
     </section>
+    <section> <h3>Permissions and user prompts</h3>
+      <p>
+        The term <a>obtain permission</a>
+        is used for acquiring trust for a given operation.
+      </p>
+      <p>
+        The term <dfn>expressed permission</dfn> refers to an act by the user, e.g.
+        via user interface or setting or host device platform features, using which
+        the user approves the permission of a <a>browsing context</a> to access the
+        given functionality.
```

Comment:
Instead of "using which" do you mean "through which"?

Reply:
fixed